src/calculator.js

```diff
@@ -136,6 +135,15 @@ function processTrends(gender, typeCol, resID, des_year) {
           case "group_4":
             cutoffs.push(item.group_4);
             cutoffsStr.push(item.group_4);
+            break;
+          case "group_5":
```

Comment:
You may not want to push anything to `cutoffs` and `cutoffsStr` in the `group_5` and `group_6` cases when the year is in the past? (because groups of 5-6 weren't allowed previously)

Reply:
Groups of 5-6 have been allowed at some points in the years of data we have, and when it's not available, it's just undefined and gets ignored by the regression model (have tested this on across multiple residences/room types/etc. for groups of 4, 5, and 6 to see if anything broke)